wftests/warrior_tests/testcases/rest_functional_tests/tc_rest_verify_content_response.xml

```diff
@@ -108,5 +108,43 @@
 			<context>positive</context>
 			<impact>impact</impact>
 		</step>
+		<step Driver="rest_driver" Keyword="perform_http_get" TS="7">
```

Comment:
Hi @venkat1077,

Please add more coverage for negative scenarios.

Thanks,
Sanika

Reply:
Added